src/controllers/user.controller.js

```diff
@@ -1,3 +1,63 @@
-exports.getProjects = () => {};
+const mongoose = require("mongoose");
+const createError = require("http-errors");
 
-exports.postProject = () => {};
+const User = require("../models/User");
+const Project = require("../models/Project");
+const ERROR = require("../constants/error");
+
+exports.getProjects = async (req, res, next) => {
+  try {
+    const { userId } = req.params;
+
+    if (!userId || !mongoose.isValidObjectId(userId)) {
+      return next(createError(400, ERROR.BAD_REQUEST));
+    }
+
+    const user = await User.findById(userId);
```

Comment:
user의 유무만 파악하고 그 데이터는 사용하지 않는 것으로 보입니다.

```suggestion
    const user = await User.exists({ _id: userId });
// -> user = { _id: userId } or null

//위 코드는 아래와 정확히 같은 작업을 수행합니다.
    const user = awiat User.findOne({ _id: userId }).select({ _id: 1 }).lean());